pkg/mover/chart.go

```diff
@@ -68,34 +68,51 @@ func (l *defaultLogger) Println(i ...interface{}) {
 	fmt.Println(i...)
 }
 
+// ChartMetadata exposes metadata about the Helm Chart to be relocated
+type ChartMetadata struct {
+	Name    string
+	Version string
+}
+
+// RegistryCredentials to access a registry
+type RegistryCredentials struct {
+	Username, Password string
+	// add other credentials field/options as needed
+}
+
+// ChartMoveRequest defines a chart move
+type ChartMoveRequest struct {
+	Chart          string
+	ImageHintsFile string
+	Rules          RewriteRules
+	FromCreds      RegistryCredentials
+	ToCreds        RegistryCredentials
```

Comment:
This is making the assumption that all of the images in the original chart are coming from the same registry.
This might not be the case if a chart uses a private registry for the main image and a public registry for a supporting image.

And that all images in the destination chart are going to the same registry. If the `--registry` is not set, we use the original registry (or registries).

Ideally, we could provide 0..n registry credentials with host, username, password combinations.

Reply:
> Ideally, we could provide 0..n registry credentials with host, username, password combinations.

Yes, I think that would be better.

We might use a `map[host]->creds`.

Reply:
There might be no need for a map, why isn't the host part of the struct, that way the property is clearly defined?